src/runtime/mprof.go

```diff
@@ -432,8 +432,8 @@ func saveblockevent(cycles, rate int64, skip int, which bucketType) {
 		b.bp().count += float64(rate) / float64(cycles)
 		b.bp().cycles += rate
 	} else {
-		b.bp().count++
-		b.bp().cycles += cycles
```

Comment:
🤔 I think this changes the output of `runtime.MutexProfile()` which was returning the unscaled mutex contention values before. Arguably that's another bug fix – but I think this needs to be explicitly called out in the commit message and perhaps even be covered by the test case?